### ISSUE TITLE
keybase chat delete-history when user reset their profile

### ISSUE THREAD

**Issue by agsimeonov** [keybase/keybase-issues#3180]
When a user (or one of multiple users in a multi-person chat) reset their profile the conversation has 2 versions and the following message displays:

```<username> reset their profile
Their encryption keys were replaced with new ones.
Jump to new conversation```

When running:
```keybase chat delete-history --private <username>```

Keybase is unable to delete this history for the old conversation.  In fact you get the following response:

```ERROR no conversation found```

**Comment by mlsteele** [comment]
Would you do a `keybase log send`?

**Comment by mlsteele** [comment]
Does `keybase chat ls` disagree with what the gui shows?

**Comment by agsimeonov** [comment]
Yes `keybase chat ls` doesn't show the conversations on the UI which are old conversation with a couple of users who reset their profile.

I did `keybase log send`  In the feedback I placed `https://github.com/keybase/keybase-issues/issues/3180`

my log id: 8367a4e6b12ecdd3eff88a1c

**Comment by mlsteele** [comment]
Thanks!

**Comment by mlsteele** [comment]
@agsimeonov I wasn't able to reproduce the problem locally. If you update your client to the latest do you still have the issues? If it's still bugging you would you do another log send and also describe in more detail what you see?

**Comment by KenUchicha** [comment]
same error
